packages/vscode-extension/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "pli-vscode",
-  "description": "VS Code extension for PL/I language support",
+  "description": "Highlighting and diagnostics for PL/I code. Powered by Zowe, hosted by the Open Mainframe Project.",
```

The extension provides quite a bit more than just highlighting and diagnostics. I would opt for a similar description to what Microsoft is using for their [TypeScript extension](https://github.com/microsoft/vscode/blob/0d17d0c5d0335bdce2ea24ed4bfec5c4e3900e1d/extensions/typescript-language-features/package.nls.json#L3C18-L3C52).

```suggestion
  "description": "Provides rich language support for PL/I code. Powered by Zowe, hosted by the Open Mainframe Project.",
```